src/components/mote/skjema/ArbeidsgiverDropdown.tsx

```diff
@@ -20,21 +23,24 @@ const ArbeidsgiverDropdown = ({
   ledere,
   label,
   id,
-}: ArbeidsgiverDropdownProps) => (
-  <Select
-    id={id}
-    label={label}
-    onChange={(e) => {
-      velgArbeidsgiver(e.target.value);
-    }}
-  >
-    <option value="VELG">{texts.chooseArbeidsgiver}</option>
-    {ledereSortertPaaNavnOgOrganisasjonsnavn(ledere).map((leder, idx) => (
-      <option value={leder.virksomhetsnummer} key={idx}>
-        {leder.virksomhetsnavn}
-      </option>
-    ))}
-  </Select>
-);
+}: ArbeidsgiverDropdownProps) =>
+  ledere.length !== 0 ? (
+    <Select
+      id={id}
+      label={label}
+      onChange={(e) => {
+        velgArbeidsgiver(e.target.value);
+      }}
+    >
+      <option value="VELG">{texts.chooseArbeidsgiver}</option>
+      {ledereSortertPaaNavnOgOrganisasjonsnavn(ledere).map((leder, idx) => (
+        <option value={leder.virksomhetsnummer} key={idx}>
+          {leder.virksomhetsnavn}
+        </option>
+      ))}
+    </Select>
+  ) : (
+    <Alertstripe type="advarsel">{texts.noArbeidsgiver}</Alertstripe>
```

Comment:
Jeg har ikke sjekket hvordan det ser ut her, men noen andre steder har vi brukt AlertstripeFullbredde for at teksten ikke skal brytes tidlig. Kan være verdt å sjekke ut her også!